scripts/launch.sh

```diff
@@ -10,6 +10,17 @@ source $SCRIPTS_DIRECTORY/install/ssh.sh
 source $SCRIPTS_DIRECTORY/install/php.sh
 source $SCRIPTS_DIRECTORY/install/nginx.sh
 
```

Comment:
Please create and load a script to load customs scripts to  `$SCRIPTS_DIRECTORY/install/custom-scripts.sh`

```
source $SCRIPTS_DIRECTORY/install/nginx.sh
source $SCRIPTS_DIRECTORY/install/custom-scripts.sh
```

Paste this to `custom-scripts.sh`

```
# If any user provided scripts are found source them
LOCALD="$SCRIPTS_DIRECTORY/local"
FILES="$( [ -e $LOCALD ] && find $LOCALD -type f ! -name '\.*' -a -name '*.sh')"
if [ -n "$FILES" ]; then
  echo "Running local customization scripts ..."
  for file in $FILES; do
    echo "-> $file ..."
    source $file
  done
fi
```